nemo/collections/asr/parts/submodules/rnnt_greedy_decoding.py

```diff
@@ -790,6 +761,29 @@ def _greedy_decode_blank_as_pad(
                                 hypotheses[kidx].timestep.append(time_idx)
                                 hypotheses[kidx].score += float(v[kidx])
                         symbols_added += 1
+                else:
```

Comment:
This `else` corresponds to the `while` loop above, right? Was that the intent? It seems unnecessary if that was the case, since we don't have any break or other way to exist the loop, except the normal violation of the loop condition, so seems like the `else` is unnecessary here and should be removed to not cause confusion

Reply:
Right. Makes sence, I'll remove `else`.

Reply:
Done